docs/getting-started-guides/fedora/flannel_multi_node_cluster.md

```diff
@@ -0,0 +1,219 @@
+
+* Note: The network interface (capable of communicating with other nodes) is commonly `eth0` but might be something else (`ens3`). Use `ip a` to list network interfaces.
+
+* Enable the flanneld service and reboot.
+
+```
+# systemctl enable flanneld
+# systemctl reboot
+```
+
+* When the system comes back up, check the interfaces on the nodes. Notice there is now a flannel.1 interface, and the ip addresses of docker0 and flannel.1 interfaces are in the same network. You will notice that docker0 is assigned a subnet (18.16.29.0/24 as shown below) on each kubernetes node out of the IP range configured above. A working output should look like this:
+
+```
+# ifconfig 
+docker0: flags=4099<UP,BROADCAST,MULTICAST>  mtu 1450
+        inet 18.16.29.1  netmask 255.255.255.0  broadcast 0.0.0.0
+        inet6 fe80::5484:7aff:fefe:9799  prefixlen 64  scopeid 0x20<link>
+        ether 56:84:7a:fe:97:99  txqueuelen 0  (Ethernet)
+        RX packets 23581  bytes 1252526 (1.1 MiB)
+        RX errors 0  dropped 0  overruns 0  frame 0
+        TX packets 40695  bytes 65342438 (62.3 MiB)
+        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
+
+eth0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
+        inet 192.168.122.77  netmask 255.255.255.0  broadcast 192.168.122.255
+        inet6 fe80::5054:ff:fe0f:6c9  prefixlen 64  scopeid 0x20<link>
+        ether 52:54:00:0f:06:c9  txqueuelen 1000  (Ethernet)
+        RX packets 180533  bytes 179117208 (170.8 MiB)
+        RX errors 0  dropped 5  overruns 0  frame 0
+        TX packets 124037  bytes 12983410 (12.3 MiB)
+        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
+
+flannel.1: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1450
+        inet 18.16.29.0  netmask 255.255.0.0  broadcast 0.0.0.0
+        inet6 fe80::e4c8:c9ff:fed7:8edf  prefixlen 64  scopeid 0x20<link>
+        ether e6:c8:c9:d7:8e:df  txqueuelen 0  (Ethernet)
+        RX packets 25  bytes 2100 (2.0 KiB)
+        RX errors 0  dropped 0  overruns 0  frame 0
+        TX packets 25  bytes 2850 (2.7 KiB)
+        TX errors 0  dropped 9 overruns 0  carrier 0  collisions 0
+
+lo: flags=73<UP,LOOPBACK,RUNNING>  mtu 65536
+        inet 127.0.0.1  netmask 255.0.0.0
+        inet6 ::1  prefixlen 128  scopeid 0x10<host>
+        loop  txqueuelen 0  (Local Loopback)
+        RX packets 0  bytes 0 (0.0 B)
+        RX errors 0  dropped 0  overruns 0  frame 0
+        TX packets 0  bytes 0 (0.0 B)
+        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
+```
+
+* From any node in the cluster, check the cluster members by issuing a query to etcd server via curl. If you set up a 1 master and 3 nodes cluster, you should see one block for each node showing the subnets they have been assigned. You can associate those subnets to each node by the MAC address (VtepMAC) and IP address (Public IP) that is listed in the output.
+
+```
+# curl -L http://fed-master:4001/v2/keys/coreos.com/network/subnets | python -mjson.tool
```

Comment:
you can control that w/ an env var

ETCDCTL_PEERS=http://fed-master:4001 etcdctl ...

or a command-line option

etcdctl --peers http://fed-master:4001 ...